api/HTMLOptionsCollection.json

```diff
@@ -112,7 +112,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": "13"
```

Comment:
This is a tricky one. The `HTMLOptionsCollection` interface was added in Edge 13, but `document.createElement('select').options.add` is in Edge 12, it was just an `HTMLCollection`. Often the difference won't matter to web developers. @Elchi3 @jpmedley please advise.

Reply:
Also tracked in https://github.com/mdn/browser-compat-data/issues/3463 now